### PR TITLE
new package: luvi, luvit, lit

### PR DESCRIPTION
**Pull request by cattokomo**
Related: #3239

### CODE REVIEW

**Comment by TomJo2000** [comment]
Continued from:
- #20462

File: packages/luvit/build.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+TERMUX_PKG_SRCURL=https://github.com/luvit/luvit/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=b792781d77028edb7e5761e96618c96162bd68747b8fced9a6fc52f123837c2c
+TERMUX_PKG_BUILD_DEPENDS="luvi"
+TERMUX_PKG_SUGGESTS="luvi, lit"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_NO_STRIP=true
+
+termux_step_configure() {
+	curl -Lo- https://github.com/luvit/lit/raw/"$(
+		. "${TERMUX_SCRIPTDIR}/packages/lit/build.sh"
```
Comment (TomJo2000) [inline packages/luvit/build.sh:15]: If you want to split this over multiple lines, please use `source` to make it a bit clearer what's going on here.
```suggestion
		source "${TERMUX_SCRIPTDIR}/packages/lit/build.sh"
```
Although I don't see why this needs to be split over multiple lines to begin with.

Comment (cattokomo) [reply packages/luvit/build.sh:15]: I'm formatting the `build.sh` with shfmt, so the splitting line probably shfmt's doing.

File: packages/luvi/build.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+TERMUX_PKG_DESCRIPTION="A project in-between luv and luvit."
+TERMUX_PKG_LICENSE="Apache-2.0"
+TERMUX_PKG_MAINTAINER="Komo @cattokomo"
+TERMUX_PKG_VERSION=20240216
+_commit=b85da58f2549a519486a7296f5b836a8f6a64880
+_version=2.14.0
+TERMUX_PKG_GIT_BRANCH=master
+TERMUX_PKG_SRCURL=git+https://github.com/luvit/luvi.git
+TERMUX_PKG_DEPENDS="pcre, openssl, luv, lua51-lpeg, libluajit"
+TERMUX_PKG_BUILD_DEPENDS="luv, lua51-lpeg"
```
Comment (TomJo2000) [inline packages/luvi/build.sh:11]: This is redundant since `luv` and `lua51-lpeg` are already dependencies.
```suggestion

```

**Review by TomJo2000 — APPROVED** [review]
I think that covers everything.
I might try to write an autoupdate function for these later.
Although I suspect that's gonna be a doozy.

**Pull request merged**